types/index.d.ts

```diff
@@ -0,0 +1,491 @@
+// Minimum TypeScript Version: 3.8
+declare module '@okta/okta-signin-widget';
+import { OktaAuth, OktaAuthOptions, Tokens } from '@okta/okta-auth-js';
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+export default class OktaSignIn implements OktaSignIn {
+  constructor(config: OktaSIW.WidgetConfig);
```

Comment:
In Enduser, we found it really easy to distinguish types by just adding in `Type` suffix to these names. Else it was getting hard to differentiate actual class names vs. types. But just a suggestion. Upto your teams' call.

Reply:
Ultimately, we intend to convert the SIW to Typescript in which case the type and the class will become one. So I think referring to the type as `OktaSignIn` will make sense.